src/content/en/tools/workbox/v3/_index.yaml

```diff
@@ -21,8 +21,8 @@ landing_page:
         Getting started with Workbox is easy, in a few minutes you'll be
         caching assets for your web site and serving them up in your web app.
       custom_image:
-        background: google-green
-        icon_name: history
+        background: orange
+        icon_name: computer
```

Comment:
Without being able to view this image, I'm notionally supportive of a switch to iconography that isn't just history :) Confirmed this change isn't staged anywhere in chat. Should be fine however.